mysql-test/suite/binlog/t/binlog_mysqlbinlog_warn_stop_datetime.test

```diff
@@ -0,0 +1,37 @@
+--echo
+--echo # MDEV-27037 mysqlbinlog emits a warning when reaching EOF before stop-datetime
+--echo
+
+--source include/have_binlog_format_statement.inc
+
+--let assert_file= $MYSQLTEST_VARDIR/tmp/warn_pos_test_file.out
```

Comment:
I think the name `assert_file` makes it seem like the file itself has content that will be validated, esp hinting towards `assert_grep.inc`. But it is actually ignored. I think the name should set the expectation that the file is ignored.

Reply:
Thanks, I made that change and added `--echo` commands that reflect what's `--exec`'d (to have parity with the `...warn_stop_position.test` file).